homeassistant/components/airvisual/sensor.py

```diff
@@ -256,32 +252,20 @@ def __init__(self, coordinator, kind, name, device_class, icon, unit):
         self._attr_icon = icon
         self._attr_unit_of_measurement = unit
         self._kind = kind
-        self._name = name
-
-    @property
-    def device_info(self):
-        """Return device registry information for this entity."""
-        return {
-            "identifiers": {(DOMAIN, self.coordinator.data["serial_number"])},
-            "name": self.coordinator.data["settings"]["node_name"],
+        self._attr_name = (
```

Comment:
Please put the attributes in alphabetical order to maintain consistency with how the integration has been done up to this point.